awscli/customizations/s3/s3handler.py

```diff
@@ -367,16 +380,14 @@ def _enqueue_upload_end_task(self, filename, upload_context):
         self.executor.submit(complete_multipart_upload_task)
 
 
-class S3StreamHandler(S3Handler):
+class S3TransferStreamHandler(BaseS3Handler):
     """
     This class is an alternative ``S3Handler`` to be used when the operation
     involves a stream since the logic is different when uploading and
     downloading streams.
     """
-    # This ensures that the number of multipart chunks waiting in the
-    # executor queue and in the threads is limited.
-    MAX_EXECUTOR_QUEUE_SIZE = 2
-    EXECUTOR_NUM_THREADS = 6
+    MAX_QUEUE_SIZE = 2
```

Comment:
I do not think we should be restraining the transfer manager to these numbers as we now properly handle memory usage with the ``max_in_memory_upload_chunks``. I think we should just keep using the normal RuntimeConfig. We even have a PR open about this: https://github.com/aws/aws-cli/pull/1973.